index.js

```diff
@@ -12,6 +12,7 @@ function fastifySwaggerUi (fastify, opts, next) {
   const staticCSP = opts.staticCSP
   const transformStaticCSP = opts.transformStaticCSP
   const hooks = opts.uiHooks
```

Comment:
I would combine all of them in a single `routeOption`

Reply:
Sure!

Reply:
Not resolved

Reply:
From what I means is the user can pass `uiRouteOption` which directly pass to the `route` registration.
You do not require separate option like `hooks` and `logLevel`

Reply:
Hi @climba03003 I really sorry about it, but it's not clear to me what the idea. Could you help me?

Reply:
Nevermind, I forgot `logLevel` is `plugin` option instead of `route` option.